tests/writemodel/writemodel.go

```diff
@@ -354,13 +354,16 @@ type ClientInterface interface {
 // for the server.
 func RunTests(t *testing.T, client ClientInterface) {
```

Comment:
For consistency, do you mind structuring this the same as our other test suites? There's no need for this to be publicly exported.

In Check, for example, we have

```
// check.go
func runTests(t *testing.T, ..., client ClientInterface) {
    ...
}    
```
and then we use that to compose the other top-level tests.